app/views/layouts/listnav/_host.html.haml

```diff
@@ -119,6 +119,15 @@
             :record_id     => @record.id,
             :title         => _("Show %{host} drift history") % {:host => host_title})
 
+        - if role_allows?(:feature => "physical_server_show") && !@record.physical_server.nil?
+          %li
+            = link_to("#{ui_lookup(:table => "physical_servers")}: #{@record.physical_server.name}",
```

Comment:
Yes, I'm aware that `:table => physical _server` seems to work but the table name is `physical_servers` and not `physical_server`. Hence, it does not seem right to use `physical_server`

The other option would be to go with `:model => 'PhysicalServer'`, which works perfectly well for singular and plural cases
`:model => 'PhysicalServer'` gives `Physical Server`
`:models => 'PhysicalServer'` gives `Physical Servers`